src/engine/strat_engine/backstore/crypt.rs

```diff
@@ -0,0 +1,476 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+use std::{
+    ffi::CString,
+    io,
+    path::{Path, PathBuf},
+};
+
+use libc::{c_void, syscall, SYS_keyctl, SYS_request_key};
+
+use libcryptsetup_rs::{
+    c_uint, CryptActivateFlags, CryptDevice, CryptInit, CryptStatusInfo, CryptVolumeKeyFlags,
+    EncryptionFormat, LibcryptErr, SafeMemHandle,
+};
+
+use crate::engine::{DevUuid, PoolUuid};
+
+type Result<T> = std::result::Result<T, LibcryptErr>;
+
+// Stratis token JSON keys
```

Comment:
Please make constants private as much as possible. We can always make them public as we find we need to.

Reply:
Now that everything has an appropriately scoped API, I think they can all be private. Fixing.